src/views/download-file.njk

```diff
@@ -0,0 +1,32 @@
+{% extends 'layout.njk' %}
+
+{% block pageTitle %}
+  {% if errorList.length > 0 %}
+    Error:
+  {% endif %}
+  Sign in
+{% endblock %}
+{% block beforeContent %}
+  {{ 
+    govukPhaseBanner(
+    {
+      tag: {
+      text: "Beta"
+    },
+    html: 'This is a new service – your <a class="govuk-link" href="#">feedback</a> will help us to improve it.'
+}) 
+}}
+{% endblock %}
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">Your document download will start soon</h1>
+      <p class="govuk-body-l">
+        <a href="{{ paths.DOWNLOAD_FILE_URI }}/data/{{ fileId }}/download" id="downloadLink" class="govuk-link">Click here to download your document</a>
+        if the download does not start in 10 seconds.</p>
+    </div>
+  </div>
+{% endblock %}
+{% block bodyEnd %}
```

Comment:
Will it override `bodyEnd` block in parent template containing GOV.UK and PIWIK scripts?

Reply:
Now that I added {{ super() }} it won't. Thanks for the spot